src/amp/components/elements/ContentAtom.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const ContentAtom: React.FC<{
+    element: ContentAtomElement;
+}> = ({}) => {
+    return <div />;
```

Comment:
Could we just return `null` to prevent anything rendering?

Reply:
Sure!